llvm/test/CodeGen/LoongArch/ir-instruction/add.ll

```diff
@@ -293,6 +295,7 @@ define i32 @add_i32_minus_0x80000000(i32 %x) {
 ; LA64-LABEL: add_i32_minus_0x80000000:
 ; LA64:       # %bb.0:
 ; LA64-NEXT:    addu16i.d $a0, $a0, -32768
+; LA64-NEXT:    addi.w $a0, $a0, 0
```

Comment:
Is this necessary?

Reply:
No. Strictly speaking, for cases where sign extension is not needed, this is a regression. In fact, it probably isn't, since almost all scenarios on LoongArch64 involve sign extension?